unpackaged/post/case_behavior_record_types/package.xml

```diff
@@ -8,5 +8,13 @@
         	<members>Case.Behavior Management</members>
         	<name>BusinessProcess</name>
 	</types>
+    <types>
```

Comment:
Any reason to not also do this for `unpackaged/post/facility_display_name`?

Reply:
ahhh good catch. I think I removed them from the original pull but forgot to add it to unmanaged. I'll see if we have a translation for it and make sure to put it in.

Reply:
I had to add placeholders.